### PR TITLE
Move all example pallets under `examples` folder.

### PR DESCRIPTION
**Pull request by jimmychu0807**
This should be a pretty trivial update.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -85,9 +85,9 @@ members = [
 	"frame/elections",
 	"frame/election-provider-multi-phase",
 	"frame/election-provider-support",
-	"frame/example",
-	"frame/example-offchain-worker",
-	"frame/example-parallel",
+	"frame/examples/basic",
+	"frame/examples/offchain-worker",
+	"frame/examples/parallel",
```
Comment (bkchr) [inline Cargo.toml:90]: Can we add some information that this is highly experimental and not really supported? Maybe in the crate level docs?

Comment (jimmychu0807) [reply Cargo.toml:89]: I updated with a note that these example pallets are not meant to be used in production in [this commit](https://github.com/paritytech/substrate/pull/10215/commits/38c52309a805c9b369b20d3b08750df2bf81094a).

**Review by bkchr — APPROVED** [review]

**Review by TriplEight — APPROVED** [review]

**Review by apopiak — APPROVED** [review]

**Comment by jimmychu0807** [comment]
I don't have right to merge the pull request and will need one of you to merge this PR @bkchr @apopiak @TriplEight . Thanks.

**Comment by bkchr** [comment]
bot merge

**Comment by NukeManDan** [comment]
Like it! Although @jimmychu0807 you broke doc links FYI - we should be aware that moving things around might do this in docs - the devhub CI will quickly spot it, but as a `nice-to-have` please consider this in future PRs.

**Comment by NukeManDan** [comment]
dumb q: are these pallets tested in CI at this level of the directory structure?

**Comment by bkchr** [comment]
> dumb q: are these pallets tested in CI at this level of the directory structure?

Yes: https://github.com/paritytech/substrate/pull/10215/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7L485

**Pull request merged**